spec/services/post_alerter_spec.rb

```diff
@@ -1413,7 +1433,7 @@ def set_topic_notification_level(user, topic, level_name)
             end
         end
 
-      expect(events.size).to eq(2)
+      expect(events.size).to eq(3)
```

Comment:
Super minor, can we check for the event names instead of just the count?

Reply:
Done!